src/utils/SOOSLogger.ts

```diff
@@ -93,12 +93,12 @@ export class SOOSLogger {
   }
 }
 
-export const globalLogger = new SOOSLogger();
+export const soosLogger = new SOOSLogger();
 
 export function setSOOSLoggerVerbose(verbose: boolean) {
```

Comment:
can we access the global rather than having these utility functions?
e.g. just do soosLogger.setVerbose directly